### PR TITLE
cleans up eth vote count repo and adapts voting output to ae script

### PR DESCRIPTION
**Pull request by kenodressel**

### CODE REVIEW

File: generate-votes-list/voting-version2.js [hunk 1]
```diff
@@ -93,30 +114,30 @@ async function setupProgressBar() {
...
-      input = fs.readFileSync("./votes.json");
+      input = fs.readFileSync("./ethVotes.json");
       json = JSON.parse(input);
     }
 }
 
 function saveJSON() {
-    let jsonString = JSON.stringify(json, Object.keys(json).concat(["vote", "totalBalance", "burnedTokens", "ownedTokens"]), 2);
+    let jsonString = JSON.stringify(json, null, 2);
     jsonString = jsonString.replace(/: "/gm, ': ')
```
Comment (thepiwo) [inline generate-votes-list/voting-version2.js]: these 3 lines should be removed, as it produces invalid json

File: generate-votes-list/voting.js [hunk 2]
```diff
@@ -94,11 +115,11 @@ async function setupProgressBar() {
 }
 
 function checkConfig() {
-    if (fs.existsSync("./.cfg") && fs.existsSync("./votes.json")) {
+    if (fs.existsSync("./.cfg") && fs.existsSync("./ethVotes.json")) {
```
Comment (thepiwo) [inline generate-votes-list/voting.js]: the ae file is `ae-votes`, so maybe `eth-votes`

File: generate-votes-list/voting-version2.js (hunk 1 above)
Comment (nikita-fuchs) [reply generate-votes-list/voting-version2.js]: I'm getting fine results with the original, and screwed json with your changes @thepiwo

File: generate-votes-list/voting.js (hunk 2 above)
Comment (nikita-fuchs) [reply generate-votes-list/voting.js]: good idea!

**Pull request merged**